### PR TITLE
Update rekordbox to 4.0.5

### PR DESCRIPTION
**Pull request by perezpaya**

### CODE REVIEW

File: Casks/rekordbox.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'rekordbox' do
-  version '3.2.2'
-  sha256 'f12cafb0764177bafbb267342c3bd367572c42cc9e340081a0e84ef2bf7ed433'
+  version '4.0.5'
+  sha256 'c74d5d099092e7fd7c2bc24920d26be836f73e34a217100b1d41cab117c40072'
 
   url "https://rekordbox.com/_app/files/Install_rekordbox_#{version.gsub('.', '_')}.pkg.zip"
```
Comment (vitorgalvao) [inline Casks/rekordbox.rb]: Do instead `url https://rekordbox.com/_app/files/Install_rekordbox_#{version.dots_to_underscores}.pkg.zip"`.

**Comment by vitorgalvao** [comment]
Looks like `sha256` is wrong. I get the same as Travis, `f7c92ad81a4c6bf942eb99e575b60012350ec58fb76a28a6d4311dda597d4d3b`. How are you calculating it?

**Comment by alexperezpaya** [comment]
I downloaded the page and executed shasum -a 256 to the file

**Comment by alexperezpaya** [comment]
Can you change it? I deleted the forked repository and I can not push code to this PR anymore

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 3804b496bb8232355dc557c5f71eb1a7641d9276. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by alexperezpaya** [comment]
Great! Thanks @vitorgalvao

**Pull request closed without merging**